vignettes/custom_appearance.Rmd

```diff
@@ -37,7 +37,8 @@ library(dplyr)
 
 It is possible to align the content by assigning `"left"`, `"center"`
 (default), and `"right"` to `.aligns` and `align` arguments in
-`in_rows()` and `rcell()`, respectively.
+`in_rows()` and `rcell()`, respectively. It is also possible to use `decimal`,
```

Comment:
We should take a sentence or two to define what decimal alignment means, and also either incorporate it into one of the existing examples in this vignette, or add a new one which showcases it.